assets/js/modules/thank-with-google/util/settings.js

```diff
@@ -84,3 +84,65 @@ export function getColorThemes() {
 		},
 	];
 }
+
+/**
+ * Gets the type value based on the buttonPlacement setting.
+ *
+ * @since n.e.x.t
+ *
+ * @param {string} buttonPlacement The buttonPlacement setting value.
+ * @return {string} "Fixed" or "Overlay" depending on if buttonPlacement is static or dynamic.
+ */
+export const getType = ( buttonPlacement ) => {
```

Comment:
Let's add simple js tests for this utility functions.

Reply:
Please check the cases where the params are falsy. Thanks!